documentation/source/user_section/tutorials/3/3-3.rst

```diff
@@ -0,0 +1,23 @@
+CSA (Per-slice)
+###############
+
+Finally, to compute CSA for individual slices, set the ``-perslice`` argument to 1, combined with the ``-z`` argument to specify slice numbers (or a range of slices).
```

Comment:
I would define here what a slice is, and how many slices there are (or how you can determine that). Also, how are the slices indexed; do they start at 0 or 1, and where is slice 0/1 located? Maybe a little infographic could help here?

Reply:
Addressed here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3094/commits/b951cfacf8343be5742d1da68117c717ef6568e8 in `documentation/source/user_section/tutorials/shape-metric-computation/csa-perslice.rst`

Looks great!